### PR TITLE
Fix command suggestions

### PR DESCRIPTION
**Pull request by Mitko-Kerezov**
As suggested by @ikoevska 
Merge after https://github.com/telerik/mobile-cli-lib/pull/451

### CODE REVIEW

**Comment by Mitko-Kerezov** [comment]
run ci

**Comment by Mitko-Kerezov** [comment]
run ci

**Comment by Mitko-Kerezov** [comment]
run ci

File: resources/command-help.json [hunk 1]
```diff
@@ -1,15 +1,19 @@
 {
-	"create|screenbuilder": "You can now go into your project directory and run `$ appbuilder add-view`.",
+	"create|screenbuilder": "Run `$ cd %s` to switch to your project directory and continue working there. Once in the project root, you can add a new view or connect to a data provider by running `$ appbuilder add-view` or `$ appbuilder add-dataprovider`, respectively.",
 
-	"create|*default": "You can now go into your project directory and run `$ appbuilder add-view`.",
+	"create|*default": "Run `$ cd %s` to switch to your project directory and continue working there. Once in the project root, you can add a new view or connect to a data provider by running `$ appbuilder add-view` or `$ appbuilder add-dataprovider`, respectively.",
 
-	"add-about": "You can check out your about page in the simulator by running `$ appbuilder simulate`",
+	"add-about": "You can now another view by running `$ appbuilder add-view`.",
```
Comment (ikoevska) [inline resources/command-help.json]: You can now ADD another view

**Comment by teobugslayer** [comment]
:+1: after green build

**Comment by Mitko-Kerezov** [comment]
run ci

**Comment by ikoevska** [comment]
:+1:  for the messages :)

**Comment by Mitko-Kerezov** [comment]
run ci

**Pull request merged**